docs/settings/fleet-settings.asciidoc

```diff
@@ -33,6 +33,9 @@ The address to use to reach the {package-manager} registry.
 The proxy address to use to reach the {package-manager} registry if an internet connection is not directly available.
 Refer to {fleet-guide}/air-gapped.html[Air-gapped environments] for details.
 
+`xpack.fleet.packageVerification.gpgKeyPath`::
+The path on disk to the GPG key used to verify {package-manager} packages. In the event that Elastic's public key
+is reissued as a security precaution, you can use this setting to specify the new key.
```

Comment:
```suggestion
The path on disk to the GPG key used to verify {package-manager} packages. If the Elastic public key
is ever reissued as a security precaution, you can use this setting to specify the new key.
```
I know we sometimes make Elastic possessive, but all the editors I've worked with in the past at other companies say this is not good from a branding perspective. :-) 

I've also tweaked the language a bit to make it clear that the public key being reissued is an unlikely event.

Reply:
Awesome thank you for this!